src/shared/components/ActionBar/ActionBar.style.ts

```diff
@@ -17,6 +17,13 @@ export const StyledActionBarContainer = styled.div`
   }
 `
 
+export const StyledInnerContainer = styled.div`
+  display: flex;
+  width: 100%;
+  max-width: var(--max-inner-width);
```

Comment:
This will have to be removed after merging #385 that handles that for all studio routes